src/trade_ui.h

```diff
@@ -46,7 +50,7 @@ class trade_selector : public inventory_drop_selector
 
     private:
         trade_ui *_parent;
-        shared_ptr_fast<ui_adaptor> _ui;
+        //shared_ptr_fast<ui_adaptor> _ui;
```

Comment:
???

Reply:
Same as below, no longer needed. Former trade_selector had multiple ui_adaptor objects, one for the header, and one for each side of the trade.

ui_adaptor as an object is initialized and handled within cataimgui::window now, if I didn't say already.